solution/zenkoversion.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: zenko.io/v1alpha1
+kind: ZenkoVersion
+metadata:
+  name: 1.1-dev
```

Comment:
should it be `2.0.0-dev`?

Reply:
Uh maybe? I am not sure where the names or versions are decided. I also am not sure if this needs a namespace, going to add it a small commit though.

Reply:
I don't think the namespace is mandatory, by default kubectl will set it in whatever is set in the kubeconfig context. I like to treat it as a deployment specific config. and it can be overwritten via `kubectl apply --namespace zenkoversion.yaml` command by the user deploying. So that each person that will use this file can deploy it in the namespace that fits. 

As per the version, it should match IMO the version of Zenko that we are targetting, today we are aiming for 2.0.0-alpha1. Something along those lines, @terrydervaux can confirm. Now, that's just my opinion, I'm open to any thoughts.

Reply:
oh yea always forget about --namespace, I was doing a sed replace approach as it sort of went inline with what kubedb was doing with envsubst. But I am just doing it for the namespace so I could use --namespace instead. Maybe when we write some docs I can swap it out.